server/genquery2/include/irods/table_column_key_maps.hpp

```diff
@@ -0,0 +1,232 @@
+#ifndef IRODS_GENQUERY2_TABLE_COLUMN_KEY_MAPS_HPP
+#define IRODS_GENQUERY2_TABLE_COLUMN_KEY_MAPS_HPP
+
+#include <map>
+#include <string_view>
+
+namespace irods::experimental::api::genquery
+{
+    struct column_info
+    {
+        std::string_view table;
+        std::string_view name;
+
+        auto operator==(const column_info& _rhs) const noexcept -> bool
+        {
+            return table == _rhs.table && name == _rhs.name;
+        }
+    }; // struct column_info
+
+    // clang-format off
```

Comment:
Is this needed?

Reply:
At the moment this is mostly for the ending curly brace, so yes - a little.

I may end up aligning each grouping for readability _(I think I had it think way wayyy back in the original implementation)_.

Reply:
Removed.